### PR TITLE
Feature/descw 1868 deployments

### PR DESCRIPTION
**Pull request by ShawnTurple**

### CODE REVIEW

File: deployments/kustomize/base/app.yaml [hunk 1]
```diff
@@ -8,24 +8,23 @@ metadata:
...
-          image: image-registry.openshift-image-registry.svc:5000/e9b785-tools/naad-app:latest
+          image: bcgovgdx/naad-app
           imagePullPolicy: Always
-          env:
-            - name: NAAD_NAME
-              value: NAAD-1
-            - name: NAAD_URL
-              value: "streaming1.naad-adna.pelmorex.com"
+          envFrom:
+          - configMapRef:
```
Comment (robrien-bcgov) [inline deployments/kustomize/base/app.yaml:20]: a cleaner way

File: Dockerfile [hunk 2]
```diff
@@ -12,6 +13,7 @@ RUN chmod +x /home/entrypoint.sh
 WORKDIR /var/www/html/
 
 COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
+USER 1001
```
Comment (kshapka-bcgov) [inline Dockerfile:16]: Does this solve some sort of permissions problem?

**Review by robrien-bcgov — APPROVED** [review]

**Review by kshapka-bcgov — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (ShawnTurple) [reply Dockerfile:16]: no, but it removes the root user, and still works, so its probably a win,

File: deployments/kustomize/base/app.yaml (hunk 1 above)
Comment (ShawnTurple) [reply deployments/kustomize/base/app.yaml:20]: if the keys in the configmap match exactly, then this is a nice way of doing it.

**Pull request merged**